worksheet_A/TerminalHacking.cpp

```diff
@@ -40,7 +63,35 @@ int main()
 	}
 
 	// TODO: implement the rest of the game
+	HackingIntro();
+
+	while (PlayerLives > 0) 
+	{
+		cout << "Please Enter Password\n";
+		//getline(cin, Guess);
+		cin >> Guess;
```

Comment:
You missed the part of the flowchart which checks that the word is in the list of options